Automattic/wp-calypso
# BackupRetention: Update backup retention disclaimer

Related to #74063

## Proposed Changes

* Update backup retention setting disclaimer

## Discussion

peaFOp-11T-p2#comment-949

## Screenshots

![image](https://user-images.githubusercontent.com/1488641/224087896-c15dfdf9-4dd0-4be2-b6b7-6eded2c8a480.png)

## Testing Instructions



* Open a Calypso live branch
* Navigate to Jetpack settings in Jetpack Cloud or Calypso Blue using a site with Jetpack VaultPress Backup plan.
* Ensure you see the new disclaimer as the screenshot provided below.

## Pre-merge Checklist
